index.html

```diff
@@ -248,17 +316,23 @@ <h1>Get started</h1>
                 <label for="email">Your email</label>
                 <input
                   type="email"
+                  onchange="storeData();"
                   id="email"
                   name="email"
                   placeholder="name@email.com"
                   required
+                  pattern="[a-z@.]+"
                 />
               </li>
             </ul>
+            <script src="valid.js"></script>
+            <!-- oninvalid="this.setCustomValidity('Enter email in lowecase')" 
+                  oninput="this.setCustomValidity('')"-->
```

Comment:
- [ ] Kindly remove this commented code. this will make your code more readable .